install.sh

```diff
@@ -41,31 +41,9 @@ function install {
   do_install
 }
 
-
```

Comment:
We cannot remove this just yet, since desktop and mobile is not in up to date with each other that causes xenial on desktop to use the old unity8 we use on mobile.